apps/server/src/modules/user-login-migration/loggable/identical-user-login-migration-system.loggable-exception.ts

```diff
@@ -0,0 +1,34 @@
+import { ErrorLogMessage, Loggable, LogMessage, ValidationErrorLogMessage } from '@src/core/logger';
+import { BusinessError } from '@shared/common';
+import { EntityId } from '@shared/domain/types';
+import { HttpStatus } from '@nestjs/common';
+
+export class IdenticalUserLoginMigrationSystemLoggableException extends BusinessError implements Loggable {
+	constructor(private readonly schoolId: string | undefined, private readonly targetSystemId: EntityId | undefined) {
```

Comment:
Those props can be undefined?

Reply:
yes, at the point where it is used. the id in the SystemDto and LegacySchoolDo are optional